api/src/main/java/edu/wpi/first/shuffleboard/api/sources/recording/serialization/Serializers.java

```diff
@@ -17,6 +19,16 @@
 
   static {
     Runtime.getRuntime().addShutdownHook(new Thread(Serializers::cleanUpAll));
```

Comment:
At a high level, can you explain why this is necessary?

Reply:
Cleans up serializers eg by closing file handles or finalizing recorded video files. Otherwise recorded video files would be unreadable

Reply:
Fair enough. So this only works if the app fails "gracefully" then (instead of via force-quit).